.github/workflows/pythonapp.yml

```diff
@@ -22,12 +22,13 @@ jobs:
     - name: Install python dev dependencies.
       run: |
         python -m pip install -U pip --no-cache-dir
-        python -m pip install pip-tools wheel
         python -m pip install -e . -r requirements.txt -r dev-requirements.txt
     - name: Lint
       run: invoke lint
     - name: Test
-      run: invoke test
+      env:
+        CODECOV_TOKEN: 282b0064-e377-478b-834c-b8de1a3ee4c4
```

Comment:
Should the token be committed or in a secret?

Reply:
It can be replaced with a secret, I would. However it's not really sensitive as the worst that can happen is someone uploads a false coverage report.